ImageFeed/Services/OAuth2TokenStorage.swift

```diff
@@ -18,9 +19,23 @@ final class OAuth2TokenStorage {
         }
         set {
             if let bearerToken = newValue {
+                keychainWrapper.removeObject(forKey: Keys.bearerToken.rawValue)
                 keychainWrapper.set(bearerToken, forKey: Keys.bearerToken.rawValue)
             }
```

Comment:
```swift
            if let bearerToken = newValue { // если newValue это nil, то условие не выполнится, значение не будет удалено, не будет добавлено
                keychainWrapper.removeObject(forKey: Keys.bearerToken.rawValue)
                keychainWrapper.set(bearerToken, forKey: Keys.bearerToken.rawValue)
            }
```

Я ожидала вот такого исправления:
```suggestion
            if let bearerToken = newValue {
                keychainWrapper.set(bearerToken, forKey: Keys.bearerToken.rawValue)
            } else {
                keychainWrapper.removeObject(forKey: Keys.bearerToken.rawValue)
            }
```

Reply:
а, это условия я не учел) спасибо)